src/output-json-flow.c

```diff
@@ -252,6 +252,10 @@ static void JsonFlowLogJSON(JsonFlowLogThread *aft, json_t *js, Flow *f)
                 json_object_set_new(hjs, "bypass",
                         json_string("capture"));
                 break;
+            case FLOW_STATE_MAP_BYPASSED:
```

Comment:
is this the only reason the FLOW_STATE_MAP_BYPASSED is added? I didn't see any other use for it so far, other than making various checks/switches more complex

Reply:
if it's purely informational, maybe we can have a cheaper solution like a special flag or var

Reply:
I did just remove this. The logging logic was not the reason I did introduce this. But it did finally appear that there was only this used. I'm not sure anyone will really care about logging so I will remove it.